internal/util/diff/pkgdiff.go

```diff
@@ -120,3 +132,28 @@ func pkgSet(pkgPath string) (sets.String, error) {
 	}
 	return pkgFiles, nil
 }
+
+// addMergeCommentToExpected copies the expected path directory contents to
+// new temp directory and adds merge comment to the resources in directory
+// it also returns the cleanup function to clean the created temp directory
+func addMergeCommentToExpected(path string) (string, func(), error) {
```

Comment:
This function seems identical to the function in `testutil`. Also, the name is confusing.

Reply:
Yes. Moved it to addmergecomments package itself. It is cleaner now.